samza-log4j/src/main/java/org/apache/samza/config/Log4jSystemConfig.java

```diff
@@ -78,7 +81,8 @@ public String getSerdeClass(String name) {
   }
 
   public String getStreamSerdeName(String systemName, String streamName) {
-    String streamSerdeNameConfig = String.format(StreamConfig.MSG_SERDE(), systemName, streamName);
-    return get(streamSerdeNameConfig, null);
+    StreamConfig streamConfig =  new StreamConfig(this);
+    scala.Option<String> option = streamConfig.getStreamMsgSerde(new SystemStream(systemName, streamName));
```

Comment:
BTW, I just realized that the logging page doesn't explicitly describe the default naming convention, so I added it. 

With this info, it should be easy for users to set properties on that stream using either the old or new stream config pattern